packages/react-native/android/src/main/java/io/invertase/notifee/NotifeeApiModule.java

```diff
@@ -300,4 +309,10 @@ public Map<String, Object> getConstants() {
     constants.put("ANDROID_API_LEVEL", android.os.Build.VERSION.SDK_INT);
     return constants;
   }
+
+  @Override
+  public boolean onRequestPermissionsResult(int requestCode, String[] permissions, int[] grantResults) {
+    return Notifee.getInstance().onRequestPermissionsResult(requestCode, permissions, grantResults);
```

Comment:
I don't think the user would need to use `onRequestPermissionsResult`. It will be invoked by `ReactNativeActivity` on receiving permission result from the user

Reply:
I'm not sure if I fully understand...

if the code is:

`const response = await notifee.requestPermission()`

`response` should be the result returned from `onRequestPermissionsResult`?